_source/logzio_collections/_log-sources/api-fetcher.md

```diff
@@ -0,0 +1,327 @@
+| http_request.method | The HTTP method. Can be GET or POST. | Required. General type only |
+| http_request.url | The Auth API url. Make sure the url is without `?` at the end. | Required. General type only | 
+| http_request.headers | Pairs of key and value the represents the headers of the HTTP request. | Optional. General type only | 
+| http_request.body | The body of the HTTP request. Will be added to HTTP POST requests only. | Optional. General type only |
+| json_paths.next_url | The json path to the next url value inside the response of the Auth API. | Required. General type only | 
+| json_paths.data | The json path to the data value inside the response of the Auth API. | Required. General type only | 
+| json_paths.data_date | The json path to the data's date value inside the response of the Auth API | Required. General type only | 
+
+##### Create a Last Start Dates text file
+
+Create an empty text file named last_start_dates.txt in the same directory as the config file:
+
+```shell
+$ touch last_start_dates.txt
+```
+
+After every successful iteration of an API, the last start date of the next iteration will be written to last_start_dates.txt. Each line starts with the API name and ends with the last start date.
+
+If you stopped the container, you can continue from the exact place you stopped, by adding the date to the API filters in the configuration.
+
+##### Run the Docker container
+
+```shell
+docker run --name logzio-api-fetcher \
+-v "$(pwd)":/app/src/shared \
+logzio/logzio-api-fetcher
+```
+
+##### Stop the Docker container
+
+When you stop the container, the code will run until the iteration is completed. To make sure it will finish the iteration on time, please give it a grace period of 30 seconds when you run the `docker stop` command.
+
+```shell
+docker stop -t 30 logzio-api-fetcher
+```
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours,
+and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs,
+see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
+
+</div>
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="oauth">
+
+## Oauth
+
+Deploy this integration to send general and Azure Graph API data using the Logz.io API fetcher. The supported types of APIs currently include:
+
+* Azure Graph
+* General (Graph)
+
+<div class="tasklist">
+
+
+##### Pull the Docker image of the Logz.io API fetcher
+
+```shell
+docker pull logzio/logzio-api-fetcher
+```
+
+
+##### Create a local directory for this integration
+
+You will need a dedicated directory to use it as mounted directory for the Docker container of the Logz.io API fetcher.
+
+```shell
+mkdir logzio-api-fetcher
+cd logzio-api-fetcher
+```
+
+##### Create a configuration file
+
+In the directory created in the previous step, create a file `config.yaml` with the following configuration:
```

Comment:
instead of 'with the following configuration', please write 'using the example configuration below'.